Fixed doc when bridge POST operation is done with empty body (#2038)
strimzi/strimzi-kafka-operator: documentation/book/con-requests-kafka-bridge.adoc

API request and response bodies are always encoded as JSON.

> * When performing consumer operations, `POST` requests must provide the following `Content-Type` header if there is a non empty body:

Reviewer: ```suggestion
* When performing consumer operations, `POST` requests must provide the following `Content-Type` header if there is a non-empty body:
```

You set the embedded data format when creating a consumer using the `consumers/_groupid_` endpoint--for more information, see the next section.

The `Content-Type` must not be set if the `POST` request has an empty body.

> It could be the case of the consumer creation operation when no configuration is provided in the body in order to create the consumer with the default values.

Reviewer 2: ```suggestion
An empty body can be used to create a consumer with the default values.
```
